functions/utils/aws.js

```diff
@@ -1,36 +1,62 @@
-const AWS = require("aws-sdk");
 const fetch = require("node-fetch");
 
-const getSignedUrl = async (aws, params) => {
-  return new Promise((resolve, reject) => {
-    const s3 = new AWS.S3({
-      region: aws.region,
-      accessKeyId: aws.key,
-      secretAccessKey: aws.secret,
-    });
-    s3.getSignedUrl("putObject", params, (err, url) => {
-    if (err) {
-      console.log("[ERROR] Error getting signed url", err);
-      reject(err)
-    }
+const getSignedUrl = async (aws, fileName, durationSeconds) => {
+  const headers = {
```

Comment:
Do you have the linter working in these files? Nit-picky but there are a mix of different quote styles used. Do you need doubles for headers?

Reply:
Not sure my linter is configured correctly, I'll fix the quotes